ci/Dockerfile

```diff
@@ -14,6 +15,9 @@ ENV SHELL=/bin/bash \
     PATH=/usr/local/cargo/bin:$PATH \
     RUST_BACKTRACE=1
 
+ARG TARGETARCH
```

Comment:
linux/amd64 emulation took forever locally on a mac, figure its useful to keep for potential multi-target images in future.